### DISCUSSION TITLE
Using Cyclops as the Base layer

### DISCUSSION THREAD

**Discussion by Grinnij** [blackears/cyclopsLevelBuilder#97] [Show and tell]
Hello, i took 2 screenshots to demonstrate how im using the current cyclops tool in my project. Hopefully they work ive not used gitub before, i just wanted to share :).

This is the blockout texture set and approach. its pretty simple for the current games restrictions and style, i made a set of about 10 1m grid textures to help separate and identify geometry as im working with it. in the screenshot some of it is stretched as i expirement with other textures and tiling.
![Godot_v4 1 2-stable_win64_apcD1Ea8TW](https://github.com/blackears/cyclopsLevelBuilder/assets/52944336/b83758a5-f3c7-49f0-b3c3-be67d2313598)

![Godot_v4 1 2-stable_win64_9tNZ4n6Vln](https://github.com/blackears/cyclopsLevelBuilder/assets/52944336/87867ea0-cc2e-404b-a6d7-c532143a7be3)

I then texture some parts of the cyclops block mesh with game ready textures, and other portions i completely cover up with geometry.
It also saves a lot of time and memory, since the majority of the levels collisions are from the cyclops blockmesh rather then needing custom unique collisions for every tiling asset or setting up collisions in-scene.

**Comment by dogman035** [comment]
Gotta ask though, why 10 textures instead of one greyscale texture that you can tint in the albedo settings?

**Reply by Grinnij** [reply]
That's a good point, i think i originally made different colored ones to quickly have available and import and get rid of what i didn't need but grayscale would have been been more versatile ahahaha. dont know why that didn't cross my mind! thanks for the tip!

**Comment by blackears** [comment]
That looks pretty good!
